### ISSUE TITLE
Accessibility: Cross-compatibility With Mainstream Windows Screen-readers Not Adequately Implemented for Task Submissions

### ISSUE THREAD

**Issue by njsch** [doubtfire-lms/doubtfire-web#484]
Currently, on Win10 & 11, it is only possible to submit assessments with Vispero/Freedom Scientific's [JAWS for Windows](https://www.freedomscientific.com/products/software/jaws/) screen-reader; however, one cannot submit assignments using NVAccess' open-source [NVDA](https://www.nvaccess.org/about-nvda/) SR.

This issue is quite easy to reproduce using a free download of [one of the later versions](https://www.nvaccess.org/download/):
1. Open OnTrack, pick a unit, have some task documentation on-hand and attempt to upload submission as 'ready for feedback'
2. When the pop-up window appears, go through the usual steps.
3. Press next and attempt to go through the agreement process on the next screen(s). When you do, you will find that the whole thing cancels even when you activate the next button. But if you refresh the screen with F5, kill the NVDA process and load JAWS into memory, it works fine if you try the same thing again.

@jakerenzella, It would probably be good to endeavour to fix this, as you'll find proportionally that international students and probably others from/with more inequitable socio-economic statuses will resort to NVDA, owing to its still relatively high functionality value in general, no cost and comparatively higher locale support (in terms of both [in-program native language support](https://github.com/nvaccess/nvda/tree/master/source/locale), as well as the very many [speech](https://github.com/espeak-ng/espeak-ng) and [Braille](https://github.com/liblouis/liblouis) localisations).

**Comment by jakerenzella** [comment]
Thanks for this information @njsch I've previously started the re-write of the upload component in our new Angular framework which should address several accessibility concerns. Knowing this is the case for the upload, I will prioritise this migration.

**Comment by njsch** [comment]
@jakerenzella, okay. Any tags/branches one could follow to know when this has been more-or-less completed?

**Comment by jakerenzella** [comment]
Here is the link to a branch on my fork: https://github.com/jakerenzella/doubtfire-web/tree/new-task-uploader
